psi4/src/psi4/cc/ccenergy/Wmbej.cc

```diff
@@ -151,21 +150,21 @@ void CCEnergyWavefunction::Wmbej_build() {
         global_dpd_->file2_mat_init(&tIA);
         global_dpd_->file2_mat_rd(&tIA);
 
-        for (int Gmb = 0; Gmb < moinfo_.nirreps; Gmb++) {
+        for (int Gmb = 0; Gmb < nirrep_; Gmb++) {
             global_dpd_->buf4_mat_irrep_row_init(&W, Gmb);
             global_dpd_->buf4_mat_irrep_row_init(&F, Gmb);
 
             for (int mb = 0; mb < F.params->rowtot[Gmb]; mb++) {
                 global_dpd_->buf4_mat_irrep_row_rd(&W, Gmb, mb);
                 global_dpd_->buf4_mat_irrep_row_rd(&F, Gmb, mb);
 
-                for (int Gj = 0; Gj < moinfo_.nirreps; Gj++) {
+                for (int Gj = 0; Gj < nirrep_; Gj++) {
                     Gf = Gj;       /* T1 is totally symmetric */
                     Ge = Gmb ^ Gf; /* <mb|fe> is totally symmetric */
 
-                    nrows = moinfo_.occpi[Gj];
-                    ncols = moinfo_.virtpi[Ge];
-                    nlinks = moinfo_.virtpi[Gf];
+                    nrows = act_occpi_[Gj];
+                    ncols = act_virpi_[Ge];
+                    nlinks = act_virpi_[Gf];
```

Comment:
quick check that `virtpi` -> `virpi` is deliberate

Reply:
Wouldn't have compiled otherwise.

Reply:
ah, yes, I was thinking module-consistent, not c++-consistent. looks like cc*/dpd/occ do "virt", while scf/dct/adc/detci/cctranssort do "vir". lgtm